pkgs/development/python-modules/capturer/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, buildPythonPackage, fetchFromGitHub, humanfriendly }:
+
+buildPythonPackage rec {
+  pname = "capturer";
+  version = "2.4";
+
+  src = fetchFromGitHub {
+    owner  = "xolox";
+    repo   = "python-capturer";
+    rev    = version;
+    sha256 = "07zy264xd0g7pz9sxjqcpwmrck334xcbb7wfss26lmvgdr5nhcb9";
+  };
+
+  checkPhase = ''
+    export PATH=$PATH:$out/bin
```

Comment:
this doesn't do anything, you overwrote the actualy check. You probably want to do a `preCheck = '' .....` instead of this.

Reply:
or to do 
```suggestion
    PATH=$PATH:$out/bin pytest .
```

Reply:
Might have to look at that. But explains why it worked wonders afterwards if it didn't run the test ;)

Reply:
Cannot get it to work with tests enabled, no clue what might be wrong. Any help is appreciated! Fixing other comments in the meantime

Reply:
this worked for me
```suggestion
  checkInputs = [ pytest ];

  checkPhase = ''
    PATH=$PATH:$out/bin pytest .
  '';
```